peripherals/rcc/rcc_l4.yaml

```
@@ -13,3 +45,44 @@ RCC:
       Range24M: [0b1001, "range 9 around 24 MHz"]
       Range32M: [0b1010, "range 10 around 32 MHz"]
       Range48M: [0b1011, "range 11 around 48 MHz"]
+
+  CCIPR:
+    ADCSEL:
+      NoClock: [0, "No clock selected"]
+      PLLSAI1: [1, "PLLSAI1 clock selected"]
+      PLLSAI2: [2, "PLLSAI2 clock selected"]
+      SYSCLK: [3, "SYSCLK clock selected"]
+    CLK48SEL:
+      HSI48: [0, "HSI48 clock selected (only for STM32L49x/L4Ax devices, otherwise no clock selected)"]
+      PLLSAI1: [1, "PLLSAI1 clock selected"]
+      PLL: [2, "PLL clock selected"]
+      MSI: [3, "MSI clock selected"]
+    "LPTIM[123]SEL":
+      PCLK: [0, "PCLK clock selected"]
+      LSI: [1, "LSI clock selected"]
+      HSI16: [2, "HSI16 clock selected"]
+      LSE: [3, "LSE clock selected"]
+    "I2C[123]SEL":
+      PCLK: [0, "PCLK clock selected"]
+      SYSCLK: [1, "SYSCLK clock selected"]
+      HSI16: [2, "HSI16 clock selected"]
+    LPUART1SEL:
+      PCLK: [0, "PCLK clock selected"]
+      SYSCLK: [1, "SYSCLK clock selected"]
+      HSI16: [2, "HSI16 clock selected"]
+      LSE: [3, "LSE clock selected"]
+    "USART[12345]SEL":
+      PCLK: [0, "PCLK clock selected"]
+      SYSCLK: [1, "SYSCLK clock selected"]
+      HSI16: [2, "HSI16 clock selected"]
+      LSE: [3, "LSE clock selected"]
+
+  BDCR:
+    RTCEN:
+      Disabled: [0, "RTC clock disabled"]
+      Enabled: [1, "RTC clock enabled"]
+    RTCSEL:
+      NoClock: [0, "No clock"]
+      LSE: [1, "LSE oscillator clock selected"]
+      LSI: [2, "LSI oscillator clock selected"]
+      HSE32: [3, "HSE oscillator clock divided by 32 selected"]
```

```suggestion
      HSE: [3, "HSE oscillator clock divided by 32 selected"]
```
I would avoid using [name][number]. That is often used to specify a frequency in MHz, which is not the case here. "HSE" is fine